enterprise/backend/src/metabase_enterprise/audit_db.clj

```diff
@@ -25,11 +26,25 @@
   []
   13371337)
 
-(defenterprise default-audit-collection-entity-id
+(def ^:private default-audit-collection-entity-id
   "Default audit collection entity (instance analytics) id."
+  "vG58R8k-QddHWA7_47umn")
+
+(def ^:private default-custom-reports-entity-id
+  "Default custom reports entity id."
+  "okNLSZKdSxaoG58JSQY54")
+
+(defenterprise default-custom-reports-id
+  "Default custom reports id."
   :feature :none
   []
-  "vG58R8k-QddHWA7_47umn")
+  (collection-entity-id->id default-custom-reports-entity-id))
+
+(defenterprise default-audit-collection-id
```

Comment:
Could you memoize the entire audit collection instead of just the ID? If the collection isn't editable it should be safe to just store it in memory indefinitely, similar to how the `admin` and `all-user` perm groups are memoized. And the ID can be trivially pulled from the collection when needed.